ts/components/TosWebviewComponent.tsx

```diff
@@ -123,22 +119,18 @@ const TosWebviewComponent: React.FunctionComponent<Props> = ({
           )}
         />
       </View>
-      {shouldRenderFooter && !hasError && (
+      {shouldRenderFooter && onAcceptTos && (
         <FooterWithButtons
-          type={"TwoButtonsInlineThird"}
-          leftButton={{
-            block: true,
-            bordered: true,
-            onPress: onExit,
-            title: I18n.t("global.buttons.exit"),
-            testID: "toSWebViewContainerFooterLeftButton"
-          }}
-          rightButton={{
-            block: true,
-            primary: true,
-            onPress: onAcceptTos,
-            title: I18n.t("onboarding.tos.accept"),
-            testID: "toSWebViewContainerFooterRightButton"
+          type="SingleButton"
+          sticky={true}
+          primary={{
+            type: "Solid",
+            buttonProps: {
+              onPress: onAcceptTos,
+              label: I18n.t("onboarding.tos.accept"),
+              accessibilityLabel: I18n.t("onboarding.tos.accept"),
```

Comment:
As above, since the translation is the same, what about putting it in a constant variable and use it instead? What do you think?